src/identity/bip44.ts

```diff
@@ -0,0 +1,258 @@
+import { mnemonicToSeedSync, validateMnemonic, wordlists } from 'bip39';
+import { BlindingDataLike } from 'liquidjs-lib';
+import { SLIP77Factory } from 'slip77';
+import { AddressInterface, IdentityType } from '../types';
+import { checkIdentityType } from '../utils';
+import { Identity, IdentityInterface, IdentityOpts } from './identity';
+import { MasterPublicKey, MasterPublicKeyOpts } from './masterpubkey';
+import { Mnemonic, MnemonicOpts } from './mnemonic';
+
+// Params needed by BIP44 address derivation
+// defines an hardened coin account derivation path
+// use "type" instead of "interface" to be compliant with identityInterface
+export type BIP44Account = {
+  cointype: number;
+  account: number;
+};
+
+export interface MasterKeyWithAccountPath {
+  masterPublicKey: MasterPublicKeyOpts['masterPublicKey'];
+  derivationPath: string;
+}
+
+export interface BIP44MasterPublicKeyOpts {
+  accounts: MasterKeyWithAccountPath[];
+  masterBlindingKey: MasterPublicKeyOpts['masterBlindingKey'];
+}
+
+export function derivationPathBIP44(cointype: number, account: number) {
+  return `m/44'/${cointype}'/${account}'`;
```

Comment:
why `44'` is hardcoded? shouldn't it depend on network? 
> 1776 as coinType for Liquid V1 and 1 for testnet,regtest etc...

Reply:
Right ! I've renamed all classes, types and files from "bip44" to "cointype". PS: I've also drop "MasterPublicKey" and replaced it by "WatchOnly" -> it makes sense to me cause several accounts mean several xpub